tests/pipelines/test_pipelines_zero_shot_object_detection.py

```diff
@@ -79,43 +79,32 @@ def test_small_model_pt(self):
 
         outputs = object_detector(
             "./tests/fixtures/tests_samples/COCO/000000039769.png",
-            text_queries=["cat", "remote", "couch"],
+            candidate_labels=["cat", "remote", "couch"],
             threshold=0.64,
         )
 
         self.assertEqual(
             nested_simplify(outputs, decimals=4),
             [
-                [
-                    {"score": 0.7235, "label": "cat", "box": {"xmin": 204, "ymin": 167, "xmax": 232, "ymax": 190}},
-                    {"score": 0.6748, "label": "remote", "box": {"xmin": 571, "ymin": 83, "xmax": 598, "ymax": 103}},
-                    {"score": 0.6456, "label": "remote", "box": {"xmin": 494, "ymin": 105, "xmax": 521, "ymax": 127}},
-                    {"score": 0.642, "label": "remote", "box": {"xmin": 67, "ymin": 274, "xmax": 93, "ymax": 297}},
-                ]
+                {"score": 0.7235, "label": "cat", "box": {"xmin": 204, "ymin": 167, "xmax": 232, "ymax": 190}},
+                {"score": 0.7218, "label": "remote", "box": {"xmin": 204, "ymin": 167, "xmax": 232, "ymax": 190}},
```

Comment:
@Narsil Something seems fishy here. First three have the exact same bounding box with different labels!

Reply:
It's a random model, I think outputs shouldn't really matter (in terms of , is it correct output?).

Unless the model structure should disallow this, then yes something would be fishy.

Reply:
As per my understanding when Non-Max Suppression happens, the highest confidence box out of overlapping boxes are selected. I am not sure if this is done within a class or across classes. Since here the bounding boxes are exactly same, I doubt if there is something wrong with post processing. I also get your point of the model being random

Reply:
Hmm now with ChunkPipeline all the labels are sent independantly.

Is the non max suppression done in the `feature_extractor` or in the model itself ?

Reply:
@Narsil @sahamrit non-max suppression is not applied anywhere yet but I can add it to the feature extractor's post-process method once this PR is merged

Reply:
In that case this PR LGTM!